pydocstringformatter/_configuration/toml_parsing.py

```diff
@@ -45,11 +45,16 @@ def parse_toml_option(
         if value is True:
             return [action.option_strings[0]]
         return []
-    if isinstance(action, argparse._StoreAction):
+    elif isinstance(action, argparse._StoreAction):
```

Comment:
```suggestion
    if isinstance(action, argparse._StoreAction):
```
There's a return in each branch in the preceding ``if``. (that's why pre-commit fail.)

Reply:
I see, I was not focusing in that error, noticed the one thrown by mypy